providers/implementations/rands/seed_src_jitter.c

```diff
@@ -290,6 +290,21 @@ static size_t jitter_get_seed(void *vseed, unsigned char **pout,
     return ret;
 }
 
+size_t ossl_rand_jitter_get_seed(unsigned char **pout, int entropy, size_t min_len, size_t max_len)
+{
+   size_t ret = 0;
+   OSSL_PARAM params[1] = { OSSL_PARAM_END };
+   PROV_JITTER *s = jitter_new(NULL, NULL, NULL);
+   if (s == NULL)
+       return ret;
```

Comment:
Thank you for PR. I wonder if should raise some error like we do in similar situation in `ossl_rand_get_entropy()`.

Reply:
True.